requirements.monitoring.txt

```diff
@@ -0,0 +1,7 @@
+numpy==1.18.2
+pandas==1.0.3
+pandas-gbq==0.13.1
+pyarrow==3.0.0
+tqdm==4.45.0
+slackclient==2.5.0
```

Comment:
apologies, the list should have been pruned. I don't think we are using `slackclient`.